lib/worker.js

```diff
@@ -63,6 +71,12 @@ class AssetComputeWorker {
 
         this.events  = new AssetComputeEvents(this.params);
         this.metrics = params.metrics || new AssetComputeMetrics(params);
+
+        // set timeout to send events before action timeout
+        if (!process.env.DISABLE_IO_EVENTS_ON_TIMEOUT) {
```

Comment:
First look at the config option. The default for a boolean option should always be false/ not set. This is the case here since IO events on timeout are to be sent by default. So the option must be "DISABLE=true".

This does lead to a double negative here in the code, but this is worth it for the consumer.

Reply:
Yep, this is the same reason we defined it this way for [`node-openwhisk-newrelic`](https://github.com/adobe/node-openwhisk-newrelic/blob/master/lib/newrelic.js#L91).

This env var is really only meant for unit testing anyways